testgrid/config.yaml

```diff
@@ -3612,6 +3633,9 @@ dashboards:
   - name: "Conformance - GCE"
     description: Runs conformance tests using kubetest against latest kubernetes release 1.11 branch on GCE
     test_group_name: ci-kubernetes-gce-conformance-latest-1-11
+  - name: "Conformance - OpenStack"
+    description: Runs conformance tests using kubetest against kubernetes v1.11 with cloud-provider-openstack
+    test_group_name: ci-periodic-cloud-provider-openstack-acceptance-test-e2e-conformance-stable-branch-v1.11
   # TODO(aishsundar) (Add 1.11 OpenStack Conformance results to release dashboard)
```

Comment:
Please remove TODO, we have got it now :) and we should add this config into **sig-release-1.11-blocking** section, see https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml#L3717